chart/templates/mayastor/priority-class/priority-class.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: scheduling.k8s.io/v1
 kind: PriorityClass
 metadata:
   name: {{ .Release.Name }}-priority-class
+  labels:
+    helm-release: {{ .Release.Name }}
```

Comment:
since you're here, can you replace mayastor on line 9 with {{ .Release.Name }} ?

Reply:
Why though? That 'mayastor' refers to the mayastor project, as in the helm chart itself.... like https://github.com/openebs/mayastor-extensions/blob/develop/chart/Chart.yaml#L2. I don't think this refers to a specific release.